### PR TITLE
Zj/remove with prose remark plugin

### PR DESCRIPTION
**Pull request by zacjones93**
The `withProse` plugin was not functioning correctly. It looks like it was a [common community solution](https://gist.github.com/samselikoff/38c59bdff8ca619a87c28e9c8d8b99f9) a while back that escapes `prose` for MDX components.  We are applying `not-prose` to our MDX components anyway so this plugin isn't needed even if it was working.

This PR removes the plugin in favor of directly applying `prose` to pages and `not-prose` to MDX components

Learn SvelteKit - Part 4 Form Actions  egghead io

g get out

### CODE REVIEW

File: src/components/mdx/article-series-list.tsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ const ArticleSeriesList: React.FC<
         {articles.map((article: any, i: number) => {
           return (
             <Link
+              key={i}
```
Comment (zacjones93) [inline src/components/mdx/article-series-list.tsx:24]: an error I found in the console

File: tailwind.config.js [hunk 2]
```diff
@@ -168,7 +168,6 @@ module.exports = {
     require('@tailwindcss/typography'),
     require('@tailwindcss/forms'),
     require('@tailwindcss/aspect-ratio'),
-    require('@tailwindcss/line-clamp'),
```
Comment (zacjones93) [inline tailwind.config.js:171]: another error being logged that was annoying. line-clamp is automatically included now

File: src/pages/blog/[slug].tsx [hunk 3]
```diff
@@ -103,7 +102,7 @@ const Tag = (props: any) => {
               </div>
             )}
           </header>
-          <main>
+          <main className="prose dark:prose-dark sm:prose-lg lg:prose-xl max-w-none dark:prose-a:text-blue-300 prose-a:text-blue-500">
```
Comment (zacjones93) [inline src/pages/blog/[slug].tsx:105]: just apply prose classes to main

Comment (zacjones93) [reply src/pages/blog/[slug].tsx:105]: `not-prose` applied to MDX components down below

``` javascript
CourseWidget: ({course: courseSlug, ...props}: any) => {
  const course = find(resources, {slug: courseSlug})
  return course ? (
    <div className="not-prose my-8">
      <CourseWidget course={course} {...props} />
    </div>
  ) : null
},
```

File: src/components/mdx/article-series-list.tsx (hunk 1 above)
Comment (joelhooks) [reply src/components/mdx/article-series-list.tsx:24]: you shouldn't use index for key in general and use article slug or similar instead
```suggestion
              key={article.slug}
```

**Pull request merged**